src/commands/createFunctionApp/stacks/models/FlexSkuModel.ts

```diff
@@ -0,0 +1,27 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+export interface Sku {
```

Comment:
If you got these types from somewhere, can you link it? If not then ignore

Reply:
Hopefully the SDK will have these properties in their release to support flex, but for now, I'm defining them based off of what I see in the payload response.